css/index.css

```diff
@@ -101,15 +142,63 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.boxes {
+    display: flex;
+    flex-wrap:wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+
+.box {
     width: 12.5%;
     height: 100px;
     background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
-    align-items: center;
     justify-content: center;
+    align-items: center;
+}
+
+.box1 {
+    background: teal;
+}
+
+.box2 {
+    background: gold;
+}
+
+.box3 {
+    background: cadetblue;
+}
+
+.box4 {
+    background: coral;
+}
+
+.box5 {
+    background: crimson;
+}
+
+.box6 {
+    background: forestgreen;
+}
+
+.box7 {
+    background: darkorchid;
+}
+
+.box8 {
+    background: hotpink;
+}
+
+.box9 {
+    background: indigo;
+}
+
+.box10 {
+    background: dodgerblue;
```

Comment:
Great job on implementing dry programming here by only applying the css property that changes, which is the color in this case, to each individual box! 👏